garrysmod/lua/includes/util/color.lua

```diff
@@ -105,3 +185,35 @@ function COLOR:ToTable()
 	return { self.r, self.g, self.b, self.a }
 
 end
+
+function COLOR:Invert()
+
+	self.r = math.abs( 255 - self.r )
+	self.g = math.abs( 255 - self.g )
+	self.b = math.abs( 255 - self.b )
+
+end
+
+function COLOR:Copy()
+
+	return Color( self.r, self.g, self.b, self.a )
+
+end
+
+function COLOR:Normalize()
+
+	self.r = self.r / 255
+	self.g = self.r / 255

```

Comment:
Shouldn't these lines be `g`, `b`, and `a` rather than `r` three more times?

Reply:
woops, typo